website/src/pages/index.tsx

```diff
@@ -0,0 +1,126 @@
+import React from "react";
+import Link from "@docusaurus/Link";
+import useDocusaurusContext from "@docusaurus/useDocusaurusContext";
+import Layout from "@theme/Layout";
+import * as icons from "../icons";
+
+import styles from "./index.module.css";
+
+type FeatureItem = {
+  title: string;
+  Icon: React.ComponentType<React.SVGProps<SVGSVGElement>>;
+  description: string;
+};
+
+const FeatureList: FeatureItem[] = [
```

Comment:
These features use inconsistent parts of speech, e.g. "self-contained"/"multilingual" vs "pub/sub logging".

Reply:
Sorry not sure what you are trying to say here

Reply:
@esthersweon might have a better way of explaining since I think this parallels what we do for blog post titles.

Reply:
It reads like this:
- MCAP is Serialization-agnostic ✅ 
- MCAP is Self-contained ✅ 
- MCAP is Multilingual ✅ 
- MCAP is Production-grade ✅ 
- MCAP is Optional compression ❓
- MCAP is High-performance writing ❓